src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -47,48 +50,106 @@ const HousingInfo = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={handleSubmit}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '8px',
+        justifyContent: 'space-between'
+        // maxWidth: '720px',
+      }}
+    >
       <div
```

Comment:
We shouldn't need to use any `<div>`s. We have Material UI components that accomplish our needs more easily and efficiently. For instance, the `<Grid>` component will help a lot with making it more responsive on all viewports (Again, see `BasicInfo`).

Reply:
@andycwilliams — the HTML `<div>`'s were already in HousingInfo.jsx with their CSS styling when I first started on this ticket, I was just going along with what was already there and not changing it too much to "not  harm the code," as we've been directed. i will now see about employing the @mui `<Grid>` and look at `BasicInfo.jsx` to see how to employ it.

Reply:
Thanks to help & coding from @joshua-cornett, `HousingInfo.jsx` now has no more standard HTML `<div>` or `<form>` elements, it now is all MUI/React `<FormSection>`, `<FormControl>`, `<Grid>`, `<TextField>`, `<Select>` and `<MenuItem>`'s, with the proper fields in the `<Select>`'s from HUD `hmis-interop`. It is now looking and working as you requested, matching the `BasicInfo.jsx` page.